bHY6yh-d29370/cod-klc89d_bHY6yh-d29370.xml

```diff
@@ -0,0 +1,414 @@
+                  </p>
+               </availability>
+            </publicationStmt>
+            <sourceDesc>
+            <listWit>
+               <witness xml:id="Ka" n="cod-klc89d">Klosterneuberg, Stiftsbibliothek, CCl. 381</witness>        
+            </listWit>
+         </sourceDesc>
+      </fileDesc>
+      <encodingDesc>
+         <schemaRef n="lbp-diplomatic-1.0.0" url="https://raw.githubusercontent.com/lombardpress/lombardpress-schema/1.0.0/src/out/diplomatic.rng"/>
+         <editorialDecl>
+            <p>Encoding of this text has followed the recommendations of the LombardPress 1.0.0
+       guidelines for a diplomatic edition.</p>
+         </editorialDecl>
+      </encodingDesc>
+      <revisionDesc status="draft">
+         <listChange>
+            <change when="2024-10-03" status="draft" n="0.0.0">
+               <p>Created file for the first time.</p>
+            </change>
+         </listChange>
+      </revisionDesc>
+   </teiHeader>
+   <text xml:lang="la">
+      <front>
+         <div xml:id="starts-on">
+            <pb ed="#Ka" n="62-r"/>
+            <cb ed="#Ka" n="a"/>
+         </div>
+      </front>
+      <body>
+         <div xml:id="bHY6yh-d29370"><!-- p2l3-r7 -->
+            <head xml:id="bHY6yh-d29370-Hd1e108">Rubrica 7</head>
+            <head xml:id="bHY6yh-d29370-Hd1e111" type="question-title">De conuersatione conspectu: societate et sollicitudine</head>
+            <p xml:id="bHY6yh-d29370-d1e114">
+              <lb ed="#Ka"/>Insatione conspectu societate et 
+              <lb ed="#Ka"/>solicitudine rubrica 7
+              <lb ed="#Ka"/><ref xml:id="bHY6yh-d29370-Rd1e114">IN genesi capitulo 2</ref> <quote xml:id="bHY6yh-d29370-Qd1e118">non est bonum homi<lb ed="#Ka" break="no"/>nem 
+              esse solum</quote> 
+            </p>
+            <p xml:id="bHY6yh-d29370-d1e126">
+              <lb ed="#Ka"/><ref xml:id="bHY6yh-d29370-Rd1e128">In leuitico numeri capitulo 20</ref> <quote xml:id="bHY6yh-d29370-Qd1e130">Quicquid detinerit 
+              <lb ed="#Ka"/>in immundum in immidus fiet</quote>
+            </p>
+            <p xml:id="bHY6yh-d29370-d1e136">
+              <lb ed="#Ka"/><ref xml:id="bHY6yh-d29370-Rd1e140">Psalmista 17 psalmus</ref> <quote xml:id="bHY6yh-d29370-Qd1e140">Cum sancto sanctus 
+              <lb ed="#Ka"/>eris et cum viro innocente in nocens 
+              <lb ed="#Ka"/>eris et cum peruerso peruerteris</quote>
+            </p>
+            <p xml:id="bHY6yh-d29370-d1e149">
+              <lb ed="#Ka"/><ref xml:id="bHY6yh-d29370-Rd1e155">Salomon in prouerbiis capitulo 13</ref> <quote xml:id="bHY6yh-d29370-Qd1e153">Qui cum 
+              <lb ed="#Ka"/>quem sapientibus graditur sapiens erit 
+              <lb ed="#Ka"/>// amicus stultorum similis efficietur //</quote>
+            </p>
+            <p xml:id="bHY6yh-d29370-d1e161">
+              <ref xml:id="bHY6yh-d29370-Rd1e167">18 capitulo</ref> 
```

Comment:
@kuriae see my change below. I don't want these trailing dbslashes to be considered part of the quote. 
I want them to be indicators of the start of the next entry. 

See below.

```suggestion
            </p>
            <p xml:id="bHY6yh-d29370-d1e149">
              <lb ed="#Ka"/><ref xml:id="bHY6yh-d29370-Rd1e155">Salomon in prouerbiis capitulo 13</ref> <quote xml:id="bHY6yh-d29370-Qd1e153">Qui cum 
              <lb ed="#Ka"/>quem sapientibus graditur sapiens erit 
              <lb ed="#Ka"/>// amicus stultorum similis efficietur</quote>
            </p>
            <p xml:id="bHY6yh-d29370-d1e161">
               // <ref xml:id="bHY6yh-d29370-Rd1e167">18 capitulo</ref> 
```